libyul/AsmAnalysis.cpp

```diff
@@ -682,7 +654,39 @@ void AsmAnalyzer::expectValidType(string const& type, SourceLocation const& _loc
 		);
 }
 
-void AsmAnalyzer::warnOnInstructions(dev::eth::Instruction _instr, SourceLocation const& _location)
+// FIXME: copy'n'paste from AsmParser. make it more general (by putting it into dev::eth namespace)
```

Comment:
A PR with "fixme - still has to be deleted" should not be in "ready for review" stage. Nobody will double-check that it was actually deleted in an upcoming PR :)